.pre-commit-config.yaml

```diff
@@ -31,8 +31,8 @@ repos:
           - --check-only
           - --diff
 
-  - repo: git://github.com/guykisel/prospector-mirror
-    rev: 7ff847e779347033ebbd9e3b88279e7f3a998b45
+  - repo: git://github.com/PyCQA/prospector
+    rev: 744112f8a837bf48af1f88e66a33e7dfe2368d85
```

Comment:
If you rebase against `master` now and drop your changes to `.pre-commit-config.yaml`, CI should work properly. :slightly_smiling_face: